partiql-ast/src/main/java/org/partiql/ast/v1/expr/ExprLit.java

```diff
@@ -0,0 +1,32 @@
+package org.partiql.ast.v1.expr;
+
+import org.jetbrains.annotations.NotNull;
+import org.partiql.ast.v1.AstNode;
+import org.partiql.ast.v1.AstVisitor;
+import org.partiql.value.PartiQLValue;
+
+import java.util.Collection;
+import java.util.Collections;
+
+/**
+ * TODO docs, equals, hashcode
+ */
+public class ExprLit extends Expr {
+    @NotNull
+    public PartiQLValue value;
```

Comment:
We've discussed this earlier, but would you like to create the literal AST representations as part of this PR? AKA:
```java
public class LiteralNumericExact extends Expr { ... }
public class LiteralNumericExactPeriod extends Expr { ... }
public class LiteralNumericApprox extends Expr { ... }
```

For:
```ebnf
<unsigned numeric literal> ::=
  <exact numeric literal>
  | <approximate numeric literal>;

<exact numeric literal> ::=
  <unsigned integer> [ <period> [ <unsigned integer> ] ]
  | <period> <unsigned integer>;

<approximate numeric literal> ::= <mantissa> E <exponent>
```

Reply:
+1 to as a follow-up for 1589